docs/docs/cli/creating-data-stores.md

```diff
@@ -68,8 +68,14 @@ spec:
 To apply the configuration, you need a [configured CLI](./configuring-your-cli.md) pointed to the instance you want to apply the data store. Then you just have to enter:
 
 ```
-tracetest data-store apply -f my/data-store/file/location.yaml
+tracetest datastore apply -f my/data-store/file/location.yaml
```

Comment:
Do we need to update this command also on [our version release blog post](https://tracetest.io/blog/v09-release-notes)?

Reply:
Yes. @kdhamric can you do it, please?

Reply:
Will need @adnanrahic to correct it...